Runtime/Extensions/CollectionsExtensions.cs

```diff
@@ -139,5 +139,21 @@ public static void EnsureListItem<T>(this IList<T> list, T item)
                 list.Add(item);
             }
         }
+
+        /// <summary>
+        /// Validate if a dictionary contains an item and add it if not found
```

Comment:
Let's fix the documentation and then it's great. We should always end sentences in doc comments with a dot and there is two empty param comments:

/// <param name="key"></param>
/// <param name="value"></param>